core/src/main/java/org/stellar/anchor/sep1/Sep1Service.java

```diff
@@ -50,6 +49,17 @@ public Sep1Service(Sep1Config sep1Config) throws IOException, InvalidConfigExcep
     }
   }
 
+  private String fetchTomlFromURL(String url) throws IOException {
```

Comment:
I don't think it's necessary to obfuscate here, right? If we throw an exception in the `Sep1Service` constructor, the Spring application would fail to initialize.

Reply:
Yea I think your right. :thumbsup: